src/FSharpSource.msbuildreferences.targets

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- F# compiler has a depedency on MsBuild.  Different VS and F# Deployments carry various MsBuild versions  -->
+  <!-- This section sets the MsBuild references the compiler requires depending on the value of the environment variable  -->
+  <!-- VisualStudioVersion, 12.0 for VS2013, 1 for VS2015 -->
+  <!-- By Default uses VS2013 values -->
+  <Choose>
+    <When Condition="'$(VisualStudioVersion)' == '14.0'">
+      <PropertyGroup>
```

Comment:
This is actually quite convenient and means we can move things forward if necessary by editing one of these two files.  Unfortunately the assembly names change for some of the assemblies as well as the version numbers.  I think this is reasonably convenient.

I would prefer not to have added two additional target files.  However, the existing two files are not used uniformly by all of the projects.  Perhaps we can fix that in the future.